src/qminer/qminer_storage.cpp

```diff
@@ -956,6 +956,77 @@ void TRecSerializator::SetFixedJsonVal(TMemBase& RecMem,
     SetFixedJsonVal(RecMem.GetBf(), RecMem.Len(), FieldSerialDesc, FieldDesc, JsonVal);
 }
 
+
+void TRecSerializator::SetFieldByte(TMem& RecMem,
```

Comment:
Can we remove duplicates by using templates for `TMem` and `TMemBase`?

Reply:
Done. Take a look.